### DISCUSSION TITLE
Add asterix * to label / placeholder / floating if the element has required rule

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/vueform#55] [Ideas]
Add asterix * to label / placeholder / floating if the element has required rule

**Comment by joosia** [comment]
This would nice or perhaps even better would be adding options for example beforeLabel and afterLabel. That way you could manually add whatever you want and style it however you wish. So with schema object it would be something like this:

`afterLabel:  '<span class="p-1 text-white bg-gray-300 rounded-sm">Required</span>'`

UPDATE:
Or actually nevermind. 🤦‍♂️ Just realised you can already do this by using the label property and to prevent the extra content from showing in error messages, just specify  fieldName separately.

**Comment by adamberecz** [comment]
Released in `1.10.4`.

It can be enabled in `vueform.config.js` or in `<Vueform>` by setting `showRequired: ['label', 'placeholder', 'floating']`. It's disabled by default.
